### PR TITLE
Remove JSR 305 / Clean-up

### PR DESCRIPTION
**Pull request by offa**
Removes JSR 305 and some minor clean-up.

-------------

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/stashNotifier/StashNotifier.java [hunk 1]
```diff
@@ -23,6 +23,7 @@
 import com.cloudbees.plugins.credentials.common.StandardListBoxModel;
 import com.cloudbees.plugins.credentials.common.UsernamePasswordCredentials;
 import com.google.inject.Injector;
+import edu.umd.cs.findbugs.annotations.NonNull;
```
Comment (scaytrase) [inline src/main/java/org/jenkinsci/plugins/stashNotifier/StashNotifier.java:26]: Note to myself. This is how it's done within Jenkins itself

https://github.com/jenkinsci/jenkins/pull/4604/files

**Review by scaytrase — APPROVED** [review]
LGTM 👍

**Comment by offa** [comment]
It seems the CI infrastructure has some issues. Builds run into the 1 h timeout even though they should take a few minutes only.

**Comment by scaytrase** [comment]
Yeah, previous PRs took long time too

**Comment by offa** [comment]
Reported: https://github.com/jenkins-infra/helpdesk/issues/2742

There was a similar problem a few days ago: https://github.com/jenkins-infra/helpdesk/issues/2733

**Comment by scaytrase** [comment]
Thanks @offa !

**Pull request merged**